core/src/main/java/org/mskcc/cbio/oncokb/util/IndicatorUtils.java

```diff
@@ -662,7 +663,7 @@ private static String getMutationEffectDescriptionForUnknownAltButWithAA(Alterat
         StringBuilder sb = new StringBuilder("");
         boolean isVus = MainUtils.isVUS(alteration);
         if (isVus) {
-            sb.append(getVUSSummary(alteration, "[[gene]] [[mutation]] [[[mutation]]]", true));
+            sb.append(getVUSSummary(getVusDate(alteration), "[[gene]] [[mutation]] [[[mutation]]]", true));
```

Comment:
What is the summary suppose to say? This seems to be failing the unit test because it says `last reviewed on gene alteration mutation`